isolates/bin/long_running_isolate.dart

```diff
@@ -0,0 +1,88 @@
+// Spawn an isolate, read multiple files, send their contents to the spawned
+// isolate, and wait for the parsed JSON.
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:isolate';
+
+final files = [
+  File('json_01.json'),
```

Comment:
🤷  – maybe make this `String` and create the `File` object below? So we don't have File instances hanging around...